packages/db/prisma/schema.prisma

```diff
@@ -40,13 +40,24 @@ model Account {
     eventGroups       EventGroup[]
     events            Event[]
     color             AccountColor @relation(fields: [colorId], references: [id])
```

Comment:
why both color and eventColor?

Reply:
I didn't bother about `color` because I needed it initially to not break the app so just added the new field. Will check how it works if I remove the old one though.

Reply:
removed it